docker-compose.yml

```
@@ -1086,6 +1087,38 @@ services:
         /arrow/ci/scripts/cpp_build.sh /arrow /build &&
         /arrow/ci/scripts/python_sdist_test.sh /arrow"
 
+  ############################ Python free-threading ##########################
+
+  ubuntu-python-313-freethreading-test:
```

Could you remove `-test` suffix like we did for `ubuntu-python`?

```suggestion
  ubuntu-python-313-freethreading:
```